quota/etcd/quotaapi/quota_server.go

```diff
@@ -40,15 +73,80 @@ func (s *Server) DeleteConfig(ctx context.Context, req *quotapb.DeleteConfigRequ
 
 // GetConfig implements quotapb.QuotaServer.GetConfig.
 func (s *Server) GetConfig(ctx context.Context, req *quotapb.GetConfigRequest) (*quotapb.Config, error) {
-	return nil, errNotImplemented
+	if req.Name == "" {
+		return nil, status.Errorf(codes.InvalidArgument, "name is required")
+	}
+
+	cfgs, err := s.qs.Configs(ctx)
+	if err != nil {
+		return nil, err
+	}
+	cfg, ok := findByName(req.Name, cfgs)
+	if !ok {
+		return nil, status.Errorf(codes.NotFound, "%q not found", req.Name)
+	}
+	return convertToAPI(cfg), nil
```

Comment:
The 5 lines above are the same as `replyFromConfigs` only with a different error class -- is it possible to commonize?